packages/lexical-table/src/LexicalTableCellNode.ts

```diff
@@ -188,6 +197,7 @@ export class TableCellNode extends ElementNode {
       colSpan: this.__colSpan,
       headerState: this.__headerState,
       rowSpan: this.__rowSpan,
+      verticalAlign: this.getVerticalAlign(),
```

Comment:
Something like this would prevent it from being serialized unless it is a non-default value:

```suggestion
      ...(isValidVerticalAlign(this.__verticalAlign) && { verticalAlign: this.__verticalAlign }),
```